code/modules/logging/log_category.dm

```diff
@@ -1,32 +1,56 @@
 /// The main datum that contains all log entries for a category
 /datum/log_category
-	/// The category this datum contains
+	/// The category name
 	var/category
+
+	/// The schema version of this log category.
+	/// Expected format of "Major.Minor.Patch"
+	var/schema_version = LOG_CATEGORY_SCHEMA_VERSION_NOT_SET
+
+	/// The master category that contains this category
+	var/datum/log_category/master_category
+
 	/// If set this config flag is checked to enable this log category
 	var/config_flag
-	/// List of all entries, in chronological order of when they were added
-	var/list/entries = list()
+
+	/// Whether or not this log should not be publically visible
+	var/secret = FALSE
+
+	/// Whether the readable version of the runtime if formatted internally instead of by rustg
```

Comment:
Did you mean `log` here, rather than `runtime`?

Reply:
```suggestion
	/// Whether the readable version of the log message is formatted internally instead of by rustg
```